modules/checkout/commerce_checkout.module

```diff
@@ -34,6 +34,13 @@ function commerce_checkout_theme() {
     'commerce_checkout_form__with_sidebar' => [
       'base hook' => 'commerce_checkout_form',
     ],
+    'commerce_checkout_order_summary' => [
+      'variables' => [
+        'order' => NULL,
```

Comment:
I know they do in Twig; but they set it via preprocess, as we do here. But it takes an "argument" of order.